static/app/components/events/autofix/autofixInsightCards.tsx

```diff
@@ -0,0 +1,354 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+import {AnimatePresence, type AnimationProps, motion} from 'framer-motion';
+
+import {Button} from 'sentry/components/button';
+import {
+  replaceHeadersWithBold,
+  SuggestedFixSnippet,
+} from 'sentry/components/events/autofix/autofixRootCause';
+import type {
+  AutofixInsight,
+  AutofixRepository,
+  BreadcrumbContext,
+} from 'sentry/components/events/autofix/types';
+import BreadcrumbItemContent from 'sentry/components/events/breadcrumbs/breadcrumbItemContent';
+import {
+  BreadcrumbIcon,
+  BreadcrumbLevel,
+  getBreadcrumbColorConfig,
+  getBreadcrumbTitle,
+} from 'sentry/components/events/breadcrumbs/utils';
+import StructuredEventData from 'sentry/components/structuredEventData';
+import Timeline from 'sentry/components/timeline';
+import {IconArrow, IconChevron, IconCode, IconFire} from 'sentry/icons';
+import {space} from 'sentry/styles/space';
+import {BreadcrumbLevelType, BreadcrumbType} from 'sentry/types/breadcrumbs';
+import {singleLineRenderer} from 'sentry/utils/marked';
+import testableTransition from 'sentry/utils/testableTransition';
+
+interface AutofixBreadcrumbSnippetProps {
+  breadcrumb: BreadcrumbContext;
+}
+
+function AutofixBreadcrumbSnippet({breadcrumb}: AutofixBreadcrumbSnippetProps) {
```

Comment:
Question: does the breadcrumb object passed into here come directly from the event?

Reply:
No, it comes from the AI in Seer.